<?xml version="1.0" encoding="UTF-8"?>

<launch>

    <!-- Launch odometry node -->
    <node name="talos_odom" pkg="talos_bauzil" type="odom_sot.py" />

    
</launch>
